src/main/kotlin/org/rust/ide/structure/RustConstTreeElement.kt

```diff
@@ -6,7 +6,7 @@ import org.rust.lang.core.psi.RustConstItemElement
 
 class RustConstTreeElement(element: RustConstItemElement) : PsiTreeElementBase<RustConstItemElement>(element) {
 
-    override fun getPresentableText(): String? = element?.name
+    override fun getPresentableText() = "${element?.name}: ${element?.type?.text}"
```

Comment:
>Is it correct?

Yep, when using `"$stringInterpolation"` it's better to assume that things may be null, because unlike in all other Kotlin code, you won't get a warning about possibly null values there. 


And let me explain a bit about this `pin` thing. Consider simplified grammar for `const` item:

```
const_item ::= CONST ident ':' type '=' expr ';'
```

What happens if a user types a part of const declaration, and invokes completion?

```
struct FooBar;

const C: Foo|
```

The parser sees `const C: Foo`, which does not match `const_item` rule, and hence we don't know that `Foo` is a part of a type, and that we should suggest `FooBar`. 

We can fix this by adding a pin attribute to the grammar:

```
const_item ::= CONST ident ':' type '=' expr ';' { pin = ident }
```

This way, we tell the parser that as soon as `const C` part is parsed, it should produce a `const_item`, even if the production as a whole does not match. But this means that the `type` field of the `RustConstItem` should be nullable: you can't omit the type in the valid Rust code, but it may be not present in an incomplete code. 

So at the moment we don't apply pin attribute to const items (and as a result completion does not work), and we have the following PSI:

```Java
public interface RustConstItemElement extends ... {
  @NotNull
  RustTypeElement getType();
  ...
}
```

And when we would add the pin, the PSI would look like this:

```Java
public interface RustConstItemElement extends ... {
  @Nullable // <- here is the change
  RustTypeElement getType();
  ...
}
```